Project.toml

```diff
@@ -13,6 +13,7 @@ KernelAbstractions = "63c18a36-062a-441e-b654-da1e3ab1ce7c"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MPI = "da04e1cc-30fd-572f-bb4f-1f8673147195"
 Oceananigans = "9e8cae18-63c1-5223-a75c-80ca9d6e9a09"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
```

Comment:
We should keep our dependencies as trim as possible - so no `Plots` in the CouplerMachine dependencies (`Pkg.rm("Plots")` in the CouplerMachine environment). If you add `Plots` from the default Julia environment (no `--project`) then it will still be accessible when you want to use it in other projects without adding it to the dependency list:

`julia -e 'using Pkg; Pkg.add("Plots")'`